packages/flutter/lib/src/gestures/pointer_signal_arena.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'events.dart';
+
+/// The callback to register with a [PointerSignalArenaManager] to express
+/// interest in a pointer signal event.
+typedef PointerSignalArenaCallback = void Function(PointerSignalEvent event);
+
+/// An arena for use with pointer signal events.
+///
+/// Objects interested in a [PointerSignalEvent] should register a callback to
+/// be called if they win the arena. Currently, the resolution is always that
+/// the first entry in the arena wins.
+///
+/// Pointer signals are immediate, so unlike a gesture arena it always resolves
+/// when the arena closes.
+class PointerSignalArenaManager {
+  final List<PointerSignalArenaCallback> _members =
+      <PointerSignalArenaCallback>[];
```

Comment:
here and in other places: generally i would avoid wrapping here. It's clearer if it's all on one line (even if that means you go a little over 80 chars).